### DISCUSSION TITLE
Y axis out by 25mm

### DISCUSSION THREAD

**Discussion by btwo5** [bassamanator/Sovol-SV06-firmware#26] [Q&A]
Hi, i know its not the cable as i ran fine on stock and hills firmware

ive flashed to klipper and whenever i home via the menu or g28 it doesnt come all the way to the front of the y axis.
when trying to bed mesh the very last row grinds as it cant go further back 

any help would be appreciated as ive spent hours on this now 

thanks

**Comment by bassamanator** [comment]
You need to start reading from [here](https://www.klipper3d.org/TMC_Drivers.html?h=driver_sgthrs#configure-printercfg-for-sensorless-homing), all the way to [Update printer.cfg with sensitivity value](https://www.klipper3d.org/TMC_Drivers.html?h=driver_sgthrs#update-printercfg-with-sensitivity-value). The idea is to determine the correct sensitivity value for sensorless homing for the y axis for your setup. Incidentally, I happen to have gone through this all over again just yesterday, it's very easy and quick. However, if you like you can use the sensitivity value set by Sovol and make the following changes:
```
[tmc2209 stepper_y]
...
driver_SGTHRS: 68
...
```
Remember to be ready to hit the emergency stop at any time.

**Comment by btwo5** [comment]
thanks for your reply,  ive done the above changing the tmc driver y  and there is no difference in where it thinks y home is

**Comment by bassamanator** [comment]
Did you restart the firmware?

**Reply by btwo5** [reply]
i tried changing the printer to 68 for the Y stepper - save config - restart klipper - ran g28. still out in the Y

i ran SET_TMC_FIELD STEPPER=stepper_y FIELD=SGTHRS VALUE=68   changed this value range - save config - restart klipper - ran g28. sitll out in the Y

i running sovol sv06 non plus

**Reply by bassamanator** [reply]
Those commands are temporary. Adjust your config as I mentioned above.

**Reply by btwo5** [reply]
think ive got it now  im at 61

**Reply by bassamanator** [reply]
Please close this discussion if resolved.

**Comment by bassamanator** [comment]
What printer?
